src/contrib/serializers/Akka.Serialization.Hyperion/reference.conf

```diff
@@ -56,6 +56,8 @@
                 #         rename-to = "System.Private.CoreLib,%core%"
                 #     }]
             }
+            
+            surrogates = []
```

Comment:
Add a HOCON comment here and link to the settings area

Reply:
Done.